pep-0496-environment-markers.rst

```diff
@@ -100,30 +120,70 @@ version numbers are done using PEP-0440 semantics.
 Strings
 -------
 
-* ``os_name``: ``os.name``
-* ``sys_platform``: ``sys.platform``
-* ``platform_release``: ``platform.release()``
-* ``implementation_name``: ``sys.implementation.name``
-* ``platform_machine``: ``platform.machine()``
-* ``platform_python_implementation``: ``platform.python_implementation()``
-
-
-If a particular string value is not available (such as ``sys.implementation.name``
-in versions of Python prior to 3.3), the corresponding marker
-variable MUST be considered equivalent to the empty string.
-
-If a particular version number value is not available (such as
-``sys.implementation.version`` in versions of Python prior to 3.3) the
-corresponding marker variable MUST be considered equivalent to ``0``
+.. list-table::
+   :header-rows: 1
+
+   * - Marker
+     - Python equivalent
+     - Sample values
+   * - ``os_name``
+     - ``os.name``
+     - ``posix``, ``java``
+   * - ``sys_platform``
+     - ``sys.platform``
+     - ``linux``, ``darwin``, ``java1.8.0_51``
+   * - ``platform_release``
+     - ``platform.release()``
+     - ``3.14.1-x86_64-linode39``, ``14.5.0``, ``1.8.0_51``
+   * - ``platform_machine``
+     - ``platform.machine()``
+     - ``x86_64``
+   * - ``platform_python_implementation``
+     - ``platform.python_implementation()``
+     - ``CPython``, ``Jython``
+   * - ``implementation_name``
+     - ``sys.implementation.name``
+     - ``cpython``
+   * - ``platform_version``
+     - ``platform.version()``
+     - ``#1 SMP Fri Apr 25 13:07:35 EDT 2014``
+
+       ``Java HotSpot(TM) 64-Bit Server VM, 25.51-b03, Oracle Corporation``
+
+       ``Darwin Kernel Version 14.5.0: Wed Jul 29 02:18:53 PDT 2015; root:xnu-2782.40.9~2/RELEASE_X86_64``
+   * - ``platform_dist_name``
+     - ``platform.dist()[0]``
+     - ``Ubuntu``
+   * - ``platform_dist_version``
+     - ``platform.dist()[1]``
+     - ``14.04``
+   * - ``platform_dist_id``
+     - ``platform.dist()[2]``
+     - ``trusty``
+
+If a particular string value is not available (such as
+``sys.implementation.name`` in versions of Python prior to 3.3, or
+``platform.dist()`` on non-Linux systems), the corresponding marker
+variable returned by setuptools will be set to the empty string.
```

Comment:
by setuptools? That presumes implementation. Just say the variable will evaluate to the empty string.